README.md

```diff
@@ -177,10 +177,11 @@ Please read the [full documentation here](https://pytorch.org/data/main/dataload
 
 ## Tutorial
 
-A tutorial of this library is [available here on the documentation site](https://pytorch.org/data/main/tutorial.html).
-It covers three topics: [using DataPipes](https://pytorch.org/data/main/tutorial.html#using-datapipes),
-[working with DataLoader](https://pytorch.org/data/main/tutorial.html#working-with-dataloader), and
-[implementing DataPipes](https://pytorch.org/data/main/tutorial.html#implementing-a-custom-datapipe).
+A tutorial of this library is
+[available here on the documentation site](https://pytorch.org/data/main/dp_tutorial.html). It covers three topics:
+[using DataPipes](https://pytorch.org/data/main/dp_tutorial.html#using-datapipes),
+[working with DataLoader](https://pytorch.org/data/main/dp_tutorial.html#working-with-dataloader), and
+[implementing DataPipes](https://pytorch.org/data/main/dp_tutorial.html#implementing-a-custom-datapipe).
```

Comment:
Can you add another section for `Using DataLoader2` with the link of https://pytorch.org/data/main/dlv2_tutorial.html